yt/visualization/mapserver/pannable_map.py

```diff
@@ -134,20 +135,19 @@ def static(self, path):
     def list_fields(self):
         d = {}
 
-        # Add deposit fields (only cic + density for now)
-        for ptype in self.ds.particle_types:
-            d[ptype] = [
-                (("deposit", f"{ptype}_cic"), False),
-                (("deposit", f"{ptype}_density"), False),
-            ]
-
         # Add fluid fields (only gas for now)
         for ftype in self.ds.fluid_types:
             d[ftype] = []
             for f in self.ds.derived_field_list:
                 if f[0] != ftype:
                     continue
-
+                # Discard fields which need ghost zones for now
+                df = self.ds.field_info[f]
+                if any([isinstance(v, ValidateSpatial) for v in df.validators]):
```

Comment:
Note that you can also write to spare two chars
```suggestion
                if any(isinstance(v, ValidateSpatial) for v in df.validators):
```
I am absolutely fine with the current line, so feel free to just close and ignore this snippet.

Reply:
Puypgrade (via pre-commit) is supposed to handle this automatically